### ISSUE TITLE
Could you care less for your users?

### ISSUE THREAD

**Issue by JJ-20** [brave/brave-browser#17964]
You say Brave p3a is optional and users arent aware that unlike chrome which respects users choice on startup, you reset thier preference with each update and enable p3a for all users either this is international or your QA tests are fake.

**Comment by bsclifton** [comment]
@deeppandya I believe we fixed this with https://github.com/brave/brave-core/pull/11063

Can you confirm?

**Comment by deeppandya** [comment]
> @deeppandya I believe we fixed this with https://github.com/brave/brave-core/pull/11063
> 
> Can you confirm?

Yes it's fixed. But just to be sure, @srirambv whenever you time, would you please check one last time ?

**Comment by srirambv** [comment]
This seems to be fixed. I can't repro the issue even when upgrading from 1.31.x to 1.36.x on nightly.
